add_to_app/android_view/app/src/main/java/dev/flutter/example/androidView/ListAdapter.kt

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package dev.flutter.example.androidView
+
+import android.content.Context
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import dev.flutter.example.androidView.databinding.AndroidCardBinding
+import io.flutter.embedding.android.FlutterView
+import io.flutter.plugin.common.MethodChannel
+import java.util.*
+import kotlin.random.Random
+
+class ListAdapter(private val context: Context, private val flutterViewEngine: FlutterViewEngine) : RecyclerView.Adapter<ListAdapter.Cell>() {
```

Comment:
Probably should have a class docstring.

Reply:
TIL Kotlin docs now does markdown and completely did away with the half tags half html stuff from javadocs. This is a huge improvement!

Reply:
done